tests/vmi_multus_test.go

```diff
@@ -60,6 +60,12 @@ const (
 	macvtapNetworkConf     = `{"apiVersion":"k8s.cni.cncf.io/v1","kind":"NetworkAttachmentDefinition","metadata":{"name":"%s","namespace":"%s", "annotations": {"k8s.v1.cni.cncf.io/resourceName": "macvtap.network.kubevirt.io/%s"}},"spec":{"config":"{ \"cniVersion\": \"0.3.1\", \"name\": \"%s\", \"type\": \"macvtap\"}"}}`
 )
 
+const (
+	sriovnet1 = "sriov"
+	sriovnet2 = "sriov2"
+	sriovnet3 = "sriov-link-enabled"
```

Comment:
what do  you  think about call the variable "sriovLinkEnabled" instead of "sriovnet3"?
it will reflect that it is differently defined.

Reply:
Check the next commit, the name changed.
The SRIOV tests need to get re-organized, but this out of scope at the moment. (move these definitions closer to usage, splitting sriov tests from multus file, etc)

Reply:
> Check the next commit, the name changed.
> The SRIOV tests need to get re-organized, but this out of scope at the moment. (move these definitions closer to usage, splitting sriov tests from multus file, etc)

OK. Sounds good.